src/model_service.cpp

```diff
@@ -198,6 +198,20 @@ Status GetModelStatusImpl::getAllModelsStatuses(std::map<std::string, tensorflow
         modelsStatusesTmp.insert({pipelineName, response});
     }
 
+    const std::vector<std::string>& mediapipeNames = manager.getMediapipeFactory().getPipelinesNames();
```

Comment:
I'd unify naming as I feel like we are mixing a little bit here. The mediapipe servables will be refered to as "pipelines", "graphs" or something else? Once we make a choice I'd go with for example:
-  mediapipePipelinesNames = manager.getMediapipeFactory().getPipelinesNames();
- mediapipeGraphsNames = manager.getMediapipeFactory().getGraphsNames();

or something else, but let's not mix terms.

Reply:
The pipeline functionality is implemented with mediapipe solution. The DAG pipeline is another implementation of the pipieline so I would stay with the name.